cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -41,6 +41,11 @@ def get_bucket_name_from_arn(bucket_arn):
     'ec2:security-group': {'label': 'EC2SecurityGroup', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
     'ec2:subnet': {'label': 'EC2Subnet', 'property': 'subnetid', 'id_func': get_short_id_from_ec2_arn},
     'ec2:vpc': {'label': 'AWSVpc', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
+    'ec2:transit-gateway': {'label': 'AWSTransitGateway', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
```

Comment:
First of all I am very very impressed you were able to use this :-)

With the suggested change regarding IDs above, we can change this to 

```
'ec2:transit-gateway': {'label': 'AWSTransitGateway', 'property': 'id',},
```

Reply:
Ha, TBH I hadn't used this API before. Took a bit of reading but it's super convenient!